BASE/Test/Microsoft.ApplicationInsights.Test/Shared/Metrics/TestUtility/StubApplicationInsightsTelemetryChannel.cs

```diff
@@ -73,5 +73,13 @@ public void Flush()
         {
             OnFlush();
         }
+
+        /// <summary>
+        /// Implements  the <see cref="ITelemetryChannel.FlushAsync" /> method.
```

Comment:
```suggestion
        /// Implements the <see cref="ITelemetryChannel.FlushAsync" /> method.
```
It'll be helpful to describe the purpose/intention of the API.